src/services/items/interfaces/item.ts

```diff
@@ -68,8 +51,9 @@ export type EmbeddedLinkItemType<S = ItemSettings> = {
 export type LocalFileItemType<S = ItemSettings> = {
   type: `${ItemType.LOCAL_FILE}`;
   extra: LocalFileItemExtra;
+  settings: FileItemSettings;
 } & Item<S>;
-export type S3FileItemType<S = ItemSettings> = {
+export type S3FileItemType<S = FileItemSettings> = {
```

Comment:
Not sure why it was done differently for the `LocalFile` and the `S3File` ?

I would prefer the S3File approach of setting the default setting type. Maybe we could even make the type not-generic like this:
```ts
export type LocalFileItemType = {
  type: `${ItemType.LOCAL_FILE}`;
  extra: LocalFileItemExtra;
} & Item<FileItemSettings>;
```
same for the s3file.